pom.xml

```diff
@@ -5,27 +5,29 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>2.9</version>
+        <version>3.56</version>
     </parent>
 
     <artifactId>docker-java-api</artifactId>
-    <version>3.0.14-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
 
     <name>Docker API Plugin</name>
     <description>Plugin providing Docker API for other plugins. Wrap docker-java and required dependency for Netty transport only. JAX-RS default transport is not usable</description>
-    <url>https://wiki.jenkins.io/display/JENKINS/Docker+Java+API+Plugin</url>
+    <url>https://plugins.jenkins.io/docker-java-api/</url>
```

Comment:
This should be the link to the docs. So GitHub or wiki

Reply:
https://wiki.jenkins.io/display/JENKINS/Docker+Java+API+Plugin is redirecting to https://plugins.jenkins.io/docker-java-api/ anyway. That's why I updated the URL.

Reply:
Yea, its not super straight forward, but the <url> is where plugins.jenkins.io will pull its content from.

see https://jenkins.io/doc/developer/publishing/documentation/

I recommend exporting the current content from wiki (using https://jenkins-wiki-exporter.jenkins.io/) then making it README.adoc inside the repo, and updating <url> to that, so it can be updated.

But i'd also split up the PRs then :)

Reply:
@MarkEWaite says:
Blog: https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/
Docs: https://jenkins.io/doc/developer/publishing/documentation/#using-github-as-a-source-of-documentation

Reply:
Ok, I see. I reverted this change.